src/components/EmptyDiary/index.tsx

```diff
@@ -0,0 +1,55 @@
+import React, { ReactElement } from 'react';
+import styled from '@emotion/styled';
+import CardEmptyBackground from '@src/assets/CardEmpty.svg';
+import Logo from '@src/assets/Logo.png';
+import Plus from '@src/assets/Plus.svg';
+
+const Wrapper = styled.div`
+  position: relative;
+  width: 280px;
+  height: 512px;
+  filter: drop-shadow(0px 4px 4px rgba(0, 0, 0, 0.25));
+
+  svg {
+    color: ${(props) => props.color};
+  }
+`;
+
+const ContentWrapper = styled.div`
+  position: absolute;
+  left: 135px;
+  top: 341px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+
+  .plus-container {
+    margin-bottom: 37px;
+    position: relative;
+    img:last-child {
+      position: absolute;
+      left: 34px;
+      top: 17px;
+    }
+    span {
+      font-family: 'Gmarket Sans';
+      font-size: 16px;
+      color: ${({ theme }) => theme.colors.white100};
+    }
+  }
+`;
+
+const EmptyCard = (
```

Comment:
```suggestion
const EmptyCard = () => (
```
함수 호출 형식으로 바꿔줘야 리액트 컴포넌트로 동작할거야!
이 부분 적용한 뒤에 index page에서 `<EmptyCard />` 를 넣으면 잘 작동할듯

Reply:
오마나~ 왜이렇게 했지